chaincfg/params.go

```diff
@@ -27,7 +26,7 @@ var (
 
 	// mainPowLimit is the highest proof of work value a ExchangeCoin block can
 	// have for the main network.  It is the value 2^224 - 1.
```

Comment:
255

Reply:
Fixed and added some comments in b2c900d